common/src/main/scala/tsec/common/VerificationStatus.scala

```diff
@@ -0,0 +1,5 @@
+package tsec.common
+
+sealed trait VerificationStatus extends Product with Serializable
+case object Verified           extends VerificationStatus
```

Comment:
I don't like the naming here, maybe this should be 
`VerificationOk` and `VerificationError` ?

Reply:
`VerificationError` sounds like something other than not passing a check though, it's basically just a boolean that will force you to check semantically what it is. `VerificationError` sounds like it could be more than one thing, but really we have no such information.